### PR TITLE
Fix expectedAction for `DELETE` method

### PR DESCRIPTION
**Pull request by JSAssassin**
Fix `expectedAction` to be `write` for `DELETE` method as well

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -96,7 +96,8 @@ export function createExpectationMiddleware({
     chosen via some other means, e.g., via the request body, then the caller
     MUST provide the expected action and not rely on default behavior. */
     if(req.ezcap.expectedAction === undefined) {
-      req.ezcap.expectedAction = req.method === 'POST' ? 'write' : 'read';
+      req.ezcap.expectedAction =
+        req.method === 'POST' || req.method === 'DELETE' ? 'write' : 'read';
```
Comment (dlongley) [inline lib/helpers.js:100]: I'm going to push a commit to change this up a bit.

**Review by dlongley — CHANGES_REQUESTED** [review]

File: lib/helpers.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/helpers.js:100]: +1, and let's include `PUT` there as well.

Comment (dlongley) [reply lib/helpers.js]: Yeah, I was already adding all common methods.

**Review by dlongley — APPROVED** [review]

**Comment by dlongley** [comment]
@JSAssassin does this look good to you? @dmitrizagidulin?

**Comment by JSAssassin** [comment]
@dlongley yup. LGTM! Will this still be a patch release though?

**Pull request merged**